src/nsls2api/services/proposal_service.py

```diff
@@ -486,10 +488,29 @@ async def generate_fake_test_proposal(
             email=fake.email(),
             bnl_id=user_bnl_id,
             username=username,
-            is_pi=is_pi,
+            is_pi=is_pi
         )
         user_list.append(user)
 
+    # Real User(s)
+    if isinstance(add_specific_user, str):
+        try:
+            person = await bnlpeople_service.get_person_by_username(add_specific_user)
+            print(person)
+            if person:
+                user = User(
+                    first_name=person.FirstName,
+                    last_name=person.LastName,
+                    email=person.BNLEmail,
+                    bnl_id=person.EmployeeNumber,
+                    username=add_specific_user,
+                    is_pi=random.choice([True, False])
```

Comment:
Good point! Since I'm focusing on only adding a single 'real user' right now I've added code in a3fe2ef771be44d33431a50bcbc8f45cd6152235 that will make them the PI if present.